src/components/dropdown/DropdownPlan.jsx

```diff
@@ -62,7 +89,7 @@ const DropdownPlan = ({ title, date, location, description, author }) => {
                             </button>
                         </div>
                     </div>
-                    <div className="flex flex-row items-center mt-3 text-base gap-2">
+                    <div className="flex sm:flex-row flex-col sm:items-center mt-3 text-base gap-4">
```

Comment:
In my opinion, I think this `gap-4` should be a `gap-2`.

Reply:
Solved